metadata_updater/metadata_updater.py

```diff
@@ -154,7 +156,7 @@ def update_metadata(dest_file, mapping):
     Note. back up taken of the original file
     """
 
-    with fileinput.FileInput(dest_file, inplace=True) as file:
+    with fileinput.FileInput(dest_file, inplace=True, encoding='utf-8') as file:
```

Comment:
looks like this started 3.10 and is causing the older github actions to fail, I am happy to require a minimum of 3.10 if you would like to remove 3.8 & 2.9 from the ci.yml